BKG/Booking.BLL/Services/Booking/IBookingService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Booking.BLL.Models;
+using Booking.BLL.Services.Generic;
+
+namespace Booking.BLL.Services.Booking
+{
+    public interface IBookingService
```

Comment:
It's better to inherit service's interface from base or generic service's interface for enhance it's functionality.